kmcd-prelude.md

```diff
@@ -229,6 +229,10 @@ module KMCD-GEN
          <kmcd-state> STATE </kmcd-state>
          <kmcd-snapshots> ... (.List => ListItem(<kmcd-state> STATE </kmcd-state>)) </kmcd-snapshots>
 
+    rule <k> (. => snapshot) ~> #assert ... </k>
+         <kmcd-snapshots> ListItem(_) </kmcd-snapshots>
+      [priority(60)]
```

Comment:
Might be good to add a comment explaining what the downstream purpose of lowering the priority here is, otherwise it's super-nonobvious to someone reading the code later on what it's doing.

Reply:
I just ended up removing the `priority(60)` attribute altogether by adding a new state `#assert-failure` which `#assert` rewrites to if it doesn't get cleared from the K cell. This way we can avoid introducing the new feature into the semantics. See 24d8558.